_data/fonts.yml

```diff
@@ -0,0 +1,12 @@
+# Font configurations
+# 'sans' is applied to navigation and body text
+# 'serif' is applied to headings
+# 'normal' defines the default font weight
+# 'bold' defines the bold font weight
+fonts:
```

Comment:
I think you should be able remove this top level fonts from this file and then it can be referenced as `site.data.fonts.sans` rather than `site.data.fonts.fonts.sans`